src/index.js

```diff
@@ -194,8 +195,61 @@ const HMMediaWeightSidebar = () => {
 		};
 	} );
 
+	const insertIframe = ( url, width = '1600px', height = '800px' ) => {
+		const iframeExists = document.getElementById( 'post-preview-iframe' );
+
+		if ( ! iframeExists ) {
+			const iframe = document.createElement( 'iframe' );
+			iframe.src = url;
+			iframe.width = width;
+			iframe.height = height;
```

Comment:
Good idea adding height in here, should help more accurately capture lazy loading / above the fold type behaviour.